Library/ViewModels/ShippingRulesViewModel.swift

```diff
@@ -32,70 +33,73 @@ public protocol ShippingRulesViewModelType {
 public final class ShippingRulesViewModel: ShippingRulesViewModelType,
   ShippingRulesViewModelInputs, ShippingRulesViewModelOutputs {
   public init() {
-    let dataInitial = Signal.combineLatest(
+    let initialData = Signal.combineLatest(
       self.viewDidLoadProperty.signal,
       self.configDataProperty.signal
     )
     .map(second)
     .skipNil()
 
-    let selectedIndex = Signal.combineLatest(
-      dataInitial,
-      self.didSelectShippingRuleAtIndexProperty.signal.skipNil()
-    )
-    .map { data, newIndex in (data.1.firstIndex(of: data.2), newIndex) }
-    .filter { oldIndex, newIndex in oldIndex != newIndex }
-    .map(second)
+    let searchText = self.searchTextDidChangeProperty.signal
+      .ksr_debounce(.milliseconds(100), on: AppEnvironment.current.scheduler)
```

Comment:
Adding a little bit of delay so that this functionality doesn't cause a bottle neck on slower devices when typing really fast.